tests/unit/test_client.py

```diff
@@ -1,89 +1,129 @@
+            callback=self,
+            api_handler=api_handler)
+
+    def result(self, data: Any, code: int) -> None:
+        print('Data: ' + str(data))
+        print('Code: ' + str(code))
+
+    def run(self, commands: Iterator[dict]):
+        self.hiro_batch_client.multi_command(commands)
+
+
 class TestClient:
-    # hiro_api_handler = PasswordAuthTokenApiHandler(
-    #     root_url='',
-    #     username='',
-    #     password='',
-    #     client_id='',
-    #     client_secret='',
-    #     secure_logging=False,
-    #     custom_endpoints={
-    #         'graph': '/api/graph/7.2'
-    #     },
-    #     headers={
-    #         'user-agent': 'TestClient'
-    #     }
-    # )
-    #
+    hiro_api_handler = PasswordAuthTokenApiHandler(
+        root_url="https://ec2-3-250-135-44.eu-west-1.compute.amazonaws.com:8443",
+        username='haas1000-connector-core',
+        password='j9dad7gond4ls2taol37ulk56%1aZ',
+        client_id='cju16o7cf0000mz77pbwbhl3q_ckqjkfc0q08r90883i7x521sy',
+        client_secret='978fa4385da282ed8190b12e9ac70ed6e65ea750f6b5282c0205a9d049913ae9f2841998c2ef13bb14db2d6cee0fd1ca9834563865b8f45c555e6ad3dd1be36a',
+        # root_url="https://core.arago.co",
+        # username='haas00019-service-dentsu-connector-dv360v2',
+        # password='pAC45246B-4A81-435F-BB92-EB35F2623036',
+        # username="haas00019-service-dentsu-haas-connector-tradedesk",
+        # password="rBqGMgclgIMN%1aZ",
+        # client_id='cju16o7cf0000mz77pbwbhl3q_ckclr7uf65usl0r821gf6t8np',
+        # client_secret='c5b8db481b41f92584b6e407b21ca95b4a5f8add53a43405551145a19708f219a01195fc3286e2d288d8c8a771e2f8ec8bffeaa0f70f9812566341622fcb3d91',
+        # username='dentsuaegis.com-arago-development',
+        # password='Mensch-Gewinne-Klinik-Eisen8ahn',
+        # username='haas00016-service-dentsu-connector-dv360v2',
+        # password='huzx0N3IMYxW%1aZ',
+        # client_id='cju16o7cg0001mz775fnoq66i_ckdyfzx3e92sk012669wa256x',
+        # client_secret='cd4b323a3d0e624f65ca5222cec432c76c163e8d1970195b93875ecd0ba27111cf8ea22b25f6fc063497421c38a103370f4b606938866c9869c6b42582d3c024',
+        secure_logging=False,
+        # custom_endpoints={
+        #     'graph': '/api/graph/7.2'
+        # },
+        # headers={
+        #     "user-agent": "TestClient"
+        # }
+        ssl_config=SSLConfig(verify=False)
+    )
+
     def test_simple_query(self):
-        pass
-        # hiro_client: HiroGraph = HiroGraph(api_handler=self.hiro_api_handler)
-        #
-        # query_result: dict = hiro_client.query('ogit\\/_type:"ogit/MARS/Machine"', limit=1, meta=True)
-        #
-        # print(query_result)
-        #
-        # assert isinstance(query_result, dict)
+        hiro_client: HiroGraph = HiroGraph(api_handler=self.hiro_api_handler)
 
-    def test_batch_command(self):
-        pass
-        # hiro_batch_client: HiroGraphBatch = HiroGraphBatch(api_handler=self.hiro_api_handler)
```

Comment:
Can we remove all the commented out code? Leave it to git history if needed again

Reply:
Those files should not have been committed in the first place. *sigh*